packages/theme-a11y/theme-a11y.js

```diff
@@ -102,11 +136,14 @@ export function focusable(container) {
  *
  * @param {Element} container - Container DOM element to trap focus inside of
  * @param {Element} elementToFocus - Element to be focused on first
+ * @param {Object} params - Settings unique to your theme
+ * @param {string} params.className - Class name to apply to element on focus.
+ * @param {string} params.ignore - CSS selector for elements to not include.
  */
 
 var trapFocusHandlers = {};
 
-export function trapFocus(container, elementToFocus) {
+export function trapFocus(container, elementToFocus, params) {
```

Comment:
This now forces users to specify `null/false` or an `Element` for `elementToFocus`. Why not add `elementToFocus` to `params`?